src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -5462,4 +5413,148 @@ public boolean isFileAccessRequest() {
     public void setFileAccessRequest(boolean fileAccessRequest) {
         this.fileAccessRequest = fileAccessRequest;
     }
+
+    public Map<String, CVoc> getCVocConf(){
+        Map <String, CVoc> cvocMap = new HashMap<>();
+        String cvocSetting = settingsService.getValueForKey(SettingsServiceBean.Key.CVocConf);
+        if (cvocSetting == null || cvocSetting.isEmpty())
+            return cvocMap;
+
+        JsonReader jsonReader = Json.createReader(new StringReader(settingsService.getValueForKey(SettingsServiceBean.Key.CVocConf)));
```

Comment:
We should either make this live inside `edu.harvard.iq.dataverse.util.json.JsonParser` (meh) or use proper [JSON-B](http://json-b.net). Or just don't use JSON, making use of MicroProfile Config API instead (which is much easier to use in non-classic deployment scenarios).

Reply:
@ekoi, something to consider.